tasks/1_golf-score/golf-score.js

```diff
@@ -0,0 +1,27 @@
+export const GOLF_SCORES = {
+  HOLE_IN_ONE: 'Hole-in-one!',
+  EAGLE: 'Eagle',
+  BIRDIE: 'Birdie',
+  PAR: 'Par',
+  BOGEY: 'Bogey',
+  DOUBLE_BOGEY: 'Double Bogey',
+  GO_HOME: 'Go Home!',
+};
+
+export function golfScore(par, strokes) {
+  if (strokes == 1) {
+    return GOLF_SCORES.HOLE_IN_ONE;
+  } else if (strokes <= par - 2 ) {
+    return GOLF_SCORES.EAGLE;
+  } else if (strokes == par - 1) {
```

Comment:
Please, use '===' instead of '=='

Reply:
Got it, changed it everywhere in my project.